src/endpoint/mod.rs

```diff
@@ -344,6 +346,27 @@ impl BidiStreamRunner {
             .get(&self.svc_name)
             .expect("service must exist at this point");
 
+        #[cfg(feature = "logger")]
+        {
+            let span = info_span!(
+                "handle",
+                "rpc.system" = "restate",
+                "rpc.service" = self.svc_name,
+                "rpc.method" = self.handler_name,
+                "replaying" = field::Empty,
```

Comment:
I'm not sure how and if this works.

`replaying` IMO should be an aspect of the individual events, not of the trace. Also what you did here https://github.com/restatedev/sdk-rust/pull/30/files#diff-6658812715089c8241f20814aaf257dbd94a5448e01cf0010a0a2da54406f844R70 i suspect it doesn't really work, because you make "replaying" a field of the whole invocation attempt.

Reply:
This is what I was looking for, But I did not realize restate-sdk-shared-core = "0.1.0"  still in the SDK.

Yes, 'replaying' field is part of the events and it is injected on state changes. This is working in my SDK. 

I will make necessary change and add a test. Not urgent. I am anyways using my fork.

Reply:
> But I did not realize restate-sdk-shared-core = "0.1.0" still in the SDK.

For testing purposes, you can change the version to a git reference: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

Reply:
It's working now though